modules/Users/Users.php

```diff
@@ -706,28 +730,6 @@ public function retrieveCurrentUserInfoFromFile($userid) {
 			}
 		}
 		$this->id = $userid;
-		$imageurl = '';
```

Comment:
recover this code, you are undoing an existing change that was added a few days ago